sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/HttpHelper.cs

```diff
@@ -68,5 +68,33 @@ internal static string GetUrl(Dictionary<string, string> tags)
 
             return string.IsNullOrWhiteSpace(url) ? null : url;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static string GetHttpStatusCode(Dictionary<string, string> tags)
+        {
+            if (tags != null && tags.TryGetValue(SemanticConventions.AttributeHttpStatusCode, out var status))
+            {
+                return status;
+            }
+
+            return "0";
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static bool GetSuccessFromHttpStatusCode(string statusCode)
+        {
+            return statusCode == "200" || statusCode == "Ok";
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static string GetHost(Dictionary<string, string> tags)
+        {
+            if (tags != null && tags.TryGetValue(SemanticConventions.AttributeHttpHost, out var host))
```

Comment:
Check this out https://github.com/open-telemetry/opentelemetry-dotnet/pull/1236.

Reply:
Will get this as a part of next PR. This changes makes this PR complex.